### PR TITLE
Add in season draft to championship show page

### LINKED ISSUE #263: Test Warnings
```
warning: variable "team_b" is unused
  test/models/championship_slot_repo_test.exs:85

warning: variable "team_c" is unused
  test/models/championship_slot_repo_test.exs:92

warning: variable "team_d" is unused
  test/models/championship_slot_repo_test.exs:99

warning: variable "team_b" is unused
  test/models/championship_result_test.exs:141

warning: variable "team_c" is unused
  test/models/championship_result_test.exs:148

warning: variable "pos_c" is unused
  test/models/championship_result_test.exs:149

warning: variable "team_d" is unused
  test/models/championship_result_test.exs:155

warning: variable "pos_d" is unused
  test/models/championship_result_test.exs:156

warning: variable "championship" is unused
  test/models/waiver_test.exs:90
```

### PR DESCRIPTION
**Pull request by axelclark**
* Add a table to show in season draft picks on championship show page
* Only shows draft table if championship has an in season draft
* Closes issue #251

* fixed issues with unused variables
* closes #263

### CODE REVIEW

File: web/admin/in_season_draft_pick.ex [hunk 1]
```diff
@@ -0,0 +1,17 @@
+defmodule Ex338.ExAdmin.InSeasonDraftPick do
```
Comment (credoci) [inline web/admin/in_season_draft_pick.ex]: Modules should have a @moduledoc tag.

File: web/models/in_season_draft_pick.ex [hunk 2]
```diff
@@ -0,0 +1,33 @@
+defmodule Ex338.InSeasonDraftPick do
```
Comment (credoci) [inline web/models/in_season_draft_pick.ex]: Modules should have a @moduledoc tag.

File: web/models/in_season_draft_pick.ex [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    belongs_to :drafted_player, Ex338.FantasyPlayer
+    belongs_to :championship, Ex338.Championship
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
```
Comment (credoci) [inline web/models/in_season_draft_pick.ex]: Parameter `struct` has same name as the `Kernel.struct` function.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/264.

**Pull request merged**